src/components/EventDetails/EventDetails.css

```diff
@@ -9,22 +9,20 @@
 .card-image {
   pointer-events: none;
   border-radius: 10px 10px 10px 10px;
-  /* height: 450px; */
   object-fit: cover;
 }
 @media (min-width: 1024px) {
-  .card_image {
+  .card-image {
```

Comment:
@arunimaChintu Why have you changed this?

Reply:
am sorry it was by mistake